test/integration/immutable-client-ip/00-install-instance.yaml

```diff
@@ -48,3 +37,15 @@ spec:
         selector:
           instance: {{ .Name }}
           label: {{ .Params.LABEL }}
+
```

Comment:
Fun fact: the order matters: `Instance` has to be applied **after** the `OperatorVerson` of the webhook will decline the CREATE request.